orko-ui/src/containers/OpenOrdersContainer.js

```diff
@@ -59,7 +59,8 @@ class OpenOrdersContainer extends React.Component {
 
 function mapStateToProps(state, props) {
   return {
-    orders: getOrdersForSelectedCoin(state)
+    orders: getOrdersForSelectedCoin(state),
+    allOrders: getOrdersForAllCoin(state)
```

Comment:
If you use the showall prop here you can avoid fetching all orders when not necessary.

These little performance tweaks add up!

Reply:
I am thinking of cache it all here so that every toggle is fast and won't trigger another fetch.

Reply:
It's already cached, and the app receives data on push from the server so it won't trigger any http requests, so the difference is minor.

This does remind me though. I think we might be missing something here. I can't tell quickly from my phone, but I think you may need to subscribe to the additional order streams on the websocket. By default, the front end only subscribes to the orders for the selected coin.

If you don't do that, you won't get updates to this panel for anything other than the selected coin (except on Coinbase which has a bug which means it ALWAYS returns all coins...)

Reply:
Aha, yes worker/socket.client.js

Reply:
Yeah, I mean another fetch from the store.
And I got what you mean for the lose of update when in all coin panel. Didn't check all the code so I miss this. Let me see what I can do here.

Reply:
Great! Thanks.

The selector uses `reselect` which already has a cache, so the only saving needed here is to avoid hitting the cache twice when you only need one set of orders.

Reply:
Ok, got it now.